include/rocksdb/env.h

```diff
@@ -318,12 +319,21 @@ class Env {
   virtual Status NewLogger(const std::string& fname,
                            shared_ptr<Logger>* result) = 0;
 
-  // Returns the number of micro-seconds since some fixed point in time. Only
-  // useful for computing deltas of time.
-  // However, it is often used as system time such as in GenericRateLimiter
+  // Returns the number of micro-seconds since some fixed point in time.
+  // It is often used as system time such as in GenericRateLimiter
   // and other places so a port needs to return system time in order to work.
   virtual uint64_t NowMicros() = 0;
 
+  // Returns the number of micro-seconds since some fixed point in time.
+  // It returns time points that are monotonic. It is used to get the inteval
+  // between time points. It is NOT susceptible to system wall clock time
+  // change.
+  virtual uint64_t NowMicrosMonotonic() {
+    return std::chrono::duration_cast<std::chrono::microseconds>(
+               std::chrono::steady_clock::now().time_since_epoch())
+        .count();
```

Comment:
Can you follow the convention to rely on specific implementation for this? Just leave this one as NowNanos() / 1000.